Void doesn't package setuptools-git, and there's no need to start now.

--- a/common_setup.py
+++ b/common_setup.py
@@ -72,6 +72,5 @@
             license='MIT license',
             platforms=['unix', 'linux'],
             cmdclass={'test': PyTest, 'egg_info': EggInfo},
-            setup_requires=['setuptools-git'],
             include_package_data=True
             )
